src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Controllers/OutgoingTransfersControllerTests.cs

```diff
@@ -476,5 +669,204 @@ public async Task SubmittedTransferPost_ShouldReturnView()
+
+            //assert
+            A.CallTo(() => mapper.Map<TransferEvidenceNotesViewModelMapTransfer, TransferEvidenceNotesViewModel>(
+                    A<TransferEvidenceNotesViewModelMapTransfer>.That.Matches(t => t.TransferAllTonnage == false &&
+                        t.TransferEvidenceNoteData ==
+                        transferEvidenceNoteData &&
+                        t.Request == null &&
+                        t.Notes.SequenceEqual(evidenceNoteData) &&
+                        t.OrganisationId == organisationId)))
+                .MustHaveHappenedOnceExactly();
+        }
+
+        [Fact]
+        public async Task EditTransferFromGet_GivenMappedModel_ModelShouldBeReturned()
+        {
+            //arrange
+            var model = TestFixture.Create<TransferEvidenceNotesViewModel>();
+            A.CallTo(() => mapper.Map<TransferEvidenceNotesViewModelMapTransfer, TransferEvidenceNotesViewModel>(
+                A<TransferEvidenceNotesViewModelMapTransfer>._)).Returns(model);
+
+            //act
+            var result =
+                await outgoingTransferEvidenceController.EditTransferFrom(organisationId, TestFixture.Create<Guid>()) as
+                    ViewResult;
+
+            //assert
+            result.Model.Should().Be(model);
+        }
+
+        [Fact]
+        public async Task EditTransferFromGet_ShouldReturnView()
+        {
+            //act
+            var result =
+                await outgoingTransferEvidenceController.EditTransferFrom(organisationId, TestFixture.Create<Guid>()) as
+                    ViewResult;
+
+            //assert
+            result.ViewName.Should().Be("EditTransferFrom");
+        }
+
+        [Fact]
+        public async Task EditTransferFromPost_GivenInvalidModel_ViewShouldBeReturned()
+        {
+            //arrange
+            var model = TestFixture.Create<TransferEvidenceNotesViewModel>();
+            outgoingTransferEvidenceController.ModelState.AddModelError("error", "error");
+
+            //act
+            var result = await outgoingTransferEvidenceController.EditTransferFrom(model) as ViewResult;
+
+            //assert
+            result.ViewName.Should().Be("EditTransferFrom");
+        }
+
+        [Fact]
+        public async Task EditTransferFromPost_GivenInvalidModel_BreadcrumbShouldBeSet()
+        {
+            //arrange
+            var model = TestFixture.Create<TransferEvidenceNotesViewModel>();
+            outgoingTransferEvidenceController.ModelState.AddModelError("error", "error");
+            var organisationName = "OrganisationName";
+
+            A.CallTo(() => cache.FetchOrganisationName(model.PcsId)).Returns(organisationName);
+
+            //act
+            await outgoingTransferEvidenceController.EditTransferFrom(model);
+
+            // assert
+            breadcrumb.ExternalOrganisation.Should().Be(organisationName);
+            breadcrumb.ExternalActivity.Should().Be(BreadCrumbConstant.SchemeManageEvidence);
```

Comment:
can you add breadcrumb.OrganisationId should be model.PcsId

Reply:
added